lib/OctokitAction.ts

```diff
@@ -68,6 +88,27 @@ export abstract class OctokitAction extends Action {
     }
   }
 
+  private async jiraTransitionRequest(method: "GET" | "POST", issueId: string, body?: any): Promise<any> {
```

Comment:
The abstraction of these layers isn't very useful. The HTTP communication needs to be extracted, to be reusable. 

We need to add `sendJiraRequest` that will do the HTTP magic. And then, similar to `sendSlackPost`: let's add two more single-line proxy methods `sendJiraPost` and `sendJiraGet`, with appropriately adjusted parameters.

`jiraTransitionRequest` will not exist as it. Instead, there should be `listTransitions` and `transitionIssue` methods that will call `sendJiraPost` or `sendJiraGet`